src/main/java/microsoft/exchange/webservices/data/AutodiscoverEndpoints.java

```diff
@@ -15,47 +15,11 @@
  */
 enum AutodiscoverEndpoints {
 
-  /**
-   * No endpoints available.
-   */
-  None(0),
+  None,
+  Legacy,
+  Soap,
+  WsSecurity,
+  WSSecuritySymmetricKey,
+  WSSecurityX509Cert;
```

Comment:
In my oppinion, comments that add no extra information, only boats the files with extra lines, are noise. I left the comments in one enum, because they included more information than the enum, but a comment /** the monday **/ for an enum "monday" just clutters the file.